### PR TITLE
Update instructions.md

### PR DESCRIPTION
**Pull request by Smitholi67**
Updated the Certification Instructions to include prereqs for participation.

Ref issues:  
https://github.com/lfn-cnti/certification/issues/131
https://github.com/lfn-cnti/certification/issues/138

### CODE REVIEW

File: instructions.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # How to submit CNTi Certification results
 
-## The tests
+## Participation Prereqs
+
+- Review the CNTi Participation Terms and Conditions (https://github.com/lfn-cnti/certification/blob/main/Certified_CNF_Terms.md)
```
Comment (lixuna) [inline instructions.md:5]:
```suggestion
- Review the CNTi Participation Terms and Conditions (https://github.com/lfn-cnti/certification/blob/main/Certified_CNTi_Terms.md)
```

**Review by lixuna — CHANGES_REQUESTED** [review]
please update URL for CNTi Terms

**Review by lixuna — APPROVED** [review]

**Review by lixuna — APPROVED** [review]

File: instructions.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # How to submit CNTi Certification results
 
-## The tests
+## Participation Prereqs
```
Comment (martin-mat) [inline instructions.md:3]: This is a formal document. Don't abbreviate.
"Participation Prerequisites"

File: instructions.md [hunk 3]
```diff
@@ -11,7 +16,7 @@ There are [## tests- TBD: ## TO BE UPDATED](docs/CNTiCertification-2.0-beta.md)
 - Observability
 - Reliability, Resilience and Availability
 
-## Prereqs
+## Technical Prereqs
```
Comment (martin-mat) [inline instructions.md:19]: Prerequisites

**Review by martin-mat — CHANGES_REQUESTED** [review]
just nitpicking about "prereqs", other than that ok.

**Review by lixuna — APPROVED** [review]

**Review by taylor — APPROVED** [review]
lgtm

**Pull request merged**